index.html

```diff
@@ -0,0 +1,62 @@
+            alt="the full microverse logo with a happy female developer in the background"
+            width="20%"
+            height="20%"
+        /></a>
+        <figcaption id="img-caption">
+          Microverse is a great place, where you have fun and meet new people
+          while learning to code and preparing to become an amazing web
+          developer
+        </figcaption>
+      </div>
+      <p id="tribute-info">
+        <b>Some Advantages of Microverse</b>
+        <li>It has very high standards</li>
+        <li>It is free until the developers get a job</li>
+        <li>It gives you a global community to work with</li>
+        <li>
+          It enables you to learn about new cultures while studying and working
+        </li>
+        <li>It is so fun and interesting</li>
+        <li>It prepares you for the global market</li>
+      </p>
+      <style>
+        body {
+          background-color: black;
+          color: white;
+          font-family: Calibri;
+        }
+        #img-caption {
+          width: 400px;
+        }
+        #image {
+          max-width: 100%;
+          height: auto;
+          display: block;
+        }
+      </style>
```

Comment:
It is better to create a separate file for your styling and link it to your HTML file with the <lin> element up in the <head>.